lua/telescope/actions/init.lua

```diff
@@ -196,8 +196,8 @@ function actions.close_pum(_)
   end
 end
 
-local do_close = function(prompt_bufnr, keepinsert)
-  local picker = action_state.get_current_picker(prompt_bufnr)
+actions._close = function(prompt_bufnr, keepinsert)
+  local picker = actions.get_current_picker(prompt_bufnr)
```

Comment:
This needs to be `action_state`

Reply:
`actions.get_current_picker` is deprecated. So can we keep `action_state.get_current_picker` here.

@elianiva After this is fixed you can merge. It also closes some issue if i recall correctly. I can't/won't be here for 2 weeks. So i hand it off to you, if that is okay :) thanks <3

Reply:
@elianiva I think this should be good now